pkg/apicast/image.go

```diff
@@ -1,6 +1,6 @@
 package apicast
 
-const currentImageVersion = "quay.io/3scale/apicast:nightly"
+const currentImageVersion = "quay.io/3scale/3scale27:apicast-3scale-2.7.0-GA"
```

Comment:
This change is not necessary.  Apicast image url should be provided in CSV metadata as env var.

PR to enable providing apicast image as env var https://github.com/3scale/apicast-operator/pull/29